src/ig-template/features/inventory/IgtInventory.ts

```diff
@@ -178,15 +176,21 @@ export class IgtInventory extends IgtFeature {
   }
 
   /**
-   * This method very inefficiently clones the inventory, and simulates adding the items see if they can be taken.
-   * It's also the only reason we're using lodash...
-   * TODO do this in a smart way.
+   * Returns if the inventory can lose the given amounts of items
```

Comment:
I think you misunderstood this (admittedly weirdly-named) method. It checked whether we could add all these items to the inventory without it overflowing

Reply:
Yeah, that is really badly named.

Reply:
Actually, it looks like IgtInventory.getSpotsLeftForItem() does that....